packages/EasyUtils/src/Math.php

```diff
@@ -76,6 +93,31 @@ public function divide(string $dividend, string $divisor, ?int $precision = null
         return $this->round($value, $precision, $mode);
     }
 
+    public function equalTo(string $rightOperand): bool
```

Comment:
What would happen if someone call this (or any other comparison method) with no call to `compareThat` before?
It looks like we have to isolate them to prevent incorrect usage.

Reply:
Maybe we should return some `MathComparisonInterface` from the `compareThat` call? This `MathComparisonInterface` could contain all the comparison methods you added.

Reply:
Yeap, I thinking about it too. Let me try it.

Reply:
Reworked